templates/nonOverlappingModules.html

```diff
@@ -0,0 +1,52 @@
+$def with (lst_of_independ_mods, lst_of_ay_sem, ay_sem)
+
+$var title:Non-Overlapping Modules
+
+<div class="container container-fluid">
+    <div class="row">
+        <div class="col-md-12">
+            <div class="row"><!--header-->
+                <h1 class="text-center"><b>Non-Overlapping Modules for $ay_sem</b></h1>
```

Comment:
![image](https://cloud.githubusercontent.com/assets/12403982/23908292/903c45e8-090e-11e7-84ea-87f819dbe038.png)
personally I find this a bit redundant, please check the preview and tell me what you think.